registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -395,6 +397,7 @@
     "sha256:c1ef711b16fe0bd082f00baacf23027c85c9c80c8390ca9ea81d57ce793bdf69": ["v1.1.5"]
     "sha256:699e47bc5f10c8ebad7132a96e26624f564328766d31b0bbdc154c8cbb2ff085": ["v1.1.6"]
     "sha256:f154f5a8244daeaed1588d149ebf8f790d419d3575e96522848ca6af6292569d": ["v1.10.0-beta.0"]
+    "sha256:ca136f3178c94e57b402b9541061afa49c53ef429abdba1b1bd7d4c110293aa3": ["v1.10.0-beta.1"]
```

Comment:
`ca136f3178c9` matches the staging repo: https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:ca136f3178c94e57b402b9541061afa49c53ef429abdba1b1bd7d4c110293aa3/details?inv=1&invt=Abs_9w 👍🏻